### PR TITLE
Jetpack plugin: update Backup, Anti-spam, and VideoPress logos

### PR DESCRIPTION
**Pull request by monsieur-z**
This PR updates the VaultPress Backup, Akismet Anti-spam, and VideoPress logos in the Jetpack plugin.

Fixes 1202858161751496-as-1203427126857118

#### Changes proposed in this Pull Request:

`js-packages/components` 
- Updated the VaultPress Backup, Akismet Anti-spam, and VideoPress icons
- Added the possibility to set the icon fill color

`plugins/jetpack`
- Updated the VaultPress Backup, Akismet Anti-spam, and VideoPress
- Updated product names for VaultPress Backup, and Akismet Anti-spam
- Deleted unused icons

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion
Context: p1HpG7-jkl-p2

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:

- Spin up a test site
- Build `plugins/jetpack` and `js-packages/components`
- Install the Jetpack plugin
- Visit the Jetpack Dashboard and notice the logos and names are updated

Screen Shot 2022-12-08 at 4 58 45 PM

- Visit the _Stats_ page and notice the logos and names are updated

Screen Shot 2022-12-08 at 4 56 46 PM

- Visit the _My Jetpack_ page and notice the logos and names are updated

Screen Shot 2022-12-08 at 4 50 54 PM

- Get a VaultPress Backup, Akismet Anti-spam, and VideoPress subscriptions
- Visit the _My Plan_ page and notice the logos and names are updated

Screen Shot 2022-12-08 at 5 04 50 PM

### CODE REVIEW

File: projects/js-packages/components/components/icons/style.module.scss [hunk 1]
```diff
@@ -1,9 +1,5 @@
 @import '@automattic/jetpack-base-styles/gutenberg-base-styles';
 
-.iconWrapper {
```
Comment (monsieur-z) [inline projects/js-packages/components/components/icons/style.module.scss:3]: Set value in the `color` prop default value instead.

**Review by manzoorwanijk — DISMISSED** [review]
Works as expected and the code looks good.

**Review by sdixon194 — DISMISSED** [review]
Not a blocker, but wondering if these logos should be aligned, I think the Akismet and Scan logos get a bit out of wack when I resize the browser (reproduced on Firefox by shrinking the browser and then expanding it again):
Screen Shot 2022-12-12 at 3 34 37 PM

**Comment by monsieur-z** [comment]
> Not a blocker, but wondering if these logos should be aligned, I think the Akismet and Scan logos get a bit out of wack when I resize the browser (reproduced on Firefox by shrinking the browser and then expanding it again):

Good catch @sdixon194! It's not as obvious on Chrome, but I've centered the icons content vertically and it seems better now.

**Comment by monsieur-z** [comment]
@sdixon194 or @manzoorwanijk, I had to rebase the PR and need another approval to be able to merge it.

**Review by sdixon194 — APPROVED** [review]

**Pull request merged**